urlnorm.py

```diff
@@ -39,6 +39,10 @@
      - more fine-grained authority parsing and normalisation
 """
 
+from __future__ import absolute_import
+from six import unichr
+import six
+from six.moves import range
```

Comment:
Could you consolidate these `six` imports?

Reply:
not sure why its done this way, but this is what `python-modernize` automatically did